Cleanup the generator boilerplate in favor of the recommended add-on driven approach (#5297)
plone/volto: packages/generator-volto/README.md

npm install -g @plone/generator-volto
```

## Compatibility

> |Version   |Volto version |

Reviewer: FYI, Markdown does not care about extra spaces in a table. I tend to avoid using extra spaces because it is maintenance work, but I also understand that extra spaces makes it prettier to the author.

```suggestion
| Plone version | Volto version |
```

  yo @plone/volto:app [<projectName>] [options]

Options:
  -h,   --help              # Print the generator's options and usage
        --skip-cache        # Do not remember prompt answers                                        Default: false
        --skip-install      # Do not automatically install dependencies                             Default: false
        --force-install     # Fail on install dependencies error                                    Default: false
        --ask-answered      # Show prompts for already configured options                           Default: false
        --volto             # Desired Volto version, if not provided, the most recent will be used
        --canary            # Desired Volto version should be a canary (alpha)                      Default: false
        --interactive       # Enable/disable interactive prompt                                     Default: true
        --skip-addons       # Don't ask for addons as part of the scaffolding

>         --addon             # Addon loader string, like: some-volto-addon:loadExtra,loadOtherExtra

Reviewer: ```suggestion
        --addon             # Add-on loader string. Example: some-volto-addon:loadExtra,loadOtherExtra
```

>         --workspace         # Yarn workspace, like: src/addons/some-volto-addon

Reviewer: ```suggestion
        --workspace         # Yarn workspace. Example: src/addons/some-volto-addon
```

        --description       # Project description

>         --defaultAddonName  # The name of the add-on project added to the project by default

Reviewer: I'm not sure what this option does. Does this option pertain to the add-on that the developer will create for the generated project? Assuming so:

```suggestion
        --defaultAddonName  # The add-on's default name to be added to the generated project.
```

Author: I'd say it's better this:

`The default add-on's name to be added to the generated project.`

added in the last commit.

Author: @stevepiercy I think it's fine as it is, the suggested text is no longer true.

Reviewer: I'm sorry, I'm not clear about what _is_ true for this option.

        --ask-answered   # Show prompts for already configured options               Default: false
        --interactive    # Enable/disable interactive prompt                         Default: true
        --template       # Use github repo template, e.g.: eea/volto-addon-template
        --outputpath     # Output path

Arguments:
  addonName  # Addon name, e.g.: @plone-collective/volto-custom-block  Type: String  Required: false
```

### Enabling an existing add-on as a theme add-on

> In case that you want that one of your add-ons became a theme, you can run this template on the top of your add-on.

Reviewer: ```suggestion
If you want one of your add-ons to be a theme, you can run this template on the top of your add-on.
```

> You should use the configuration option `outputpath` to pass the path of your add-on.

Reviewer: ```suggestion
Use the configuration option `outputpath` for the path of your add-on.
```

> Provided your add-on is located at `./testadon` folder:

Reviewer: ```suggestion
Assuming your add-on is located at `./testaddon` folder, you would issue the following shell command.
```
